### PR TITLE
add un/archive workflow

### LINKED ISSUE #109: Support for (un)archiving feedstocks
Would be useful to have an admin request here for handling archiving/unarchiving feedstocks.

### PR DESCRIPTION
**Pull request by jaimergp**
Closes #109

### CODE REVIEW

**Comment by beckermr** [comment]
We should update the readme on the repo. Otherwise looks great.

**Comment by jaimergp** [comment]
Oops, you are right. I also updated the PR template :)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -26,14 +29,21 @@ What will happen when a package is marked broken?
...
+
+* [ ] I want to mark a package as broken (or not broken):
+  * [ ] Make sure your package is in the right spot (`broken/*` for adding the
+    `broken` label, `not_broken/*` for removing the `broken` label, or `token_reset/*`
+    for token resets)
+  * [ ] Added a description of the problem with the package in the PR description.
+  * [ ] Pinged the team for the package for their input.
+
+* [ ] I want to archive a feedstock:
+  * [ ] The maintainers of that feedstock have agreed and consented in this PR.
```
Comment (jakirkham) [inline .github/PULL_REQUEST_TEMPLATE.md:43]: Maybe we should borrow the line from above to be more instructive

```suggestion
  * [ ] Pinged the team for that feedstock for their input
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -26,14 +29,21 @@ What will happen when a package is marked broken?
...
+  * [ ] Make sure you have opened an issue explaining why the feedstock was archived.
```
Comment (jakirkham) [inline .github/PULL_REQUEST_TEMPLATE.md:44]: Clarify issue location

```suggestion
  * [ ] Make sure you have opened an issue on the feedstock explaining why it was archived.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -26,14 +29,21 @@ What will happen when a package is marked broken?
...
+  * [ ] Linked that issue in this PR description.
+  * [ ] Added links to any relevant issues/PRs in the PR description.
```
Comment (jakirkham) [inline .github/PULL_REQUEST_TEMPLATE.md:46]: Not sure if item is needed in the checklist

```suggestion
  * [ ] Added links to any other relevant issues/PRs in the PR description.
```

**Comment by jakirkham** [comment]
Really like this change. Also should make it easier to figure out why something was archived and revert if needed

File: README.md [hunk 4]
```diff
@@ -38,3 +38,16 @@ for which the label `broken` will be removed. See `not_broken/example.txt` for a
...
+
+If you want to request a feedstock to be archived, place the name of your feedstock in a
+new `.txt` file in `archive/`. See `archive/example.txt` for an example. You 
+should use the name without the `-feedstock` suffix (e.g., for `python-feedstock`, you put in just `python`).
+
+Before a feedstock is archived, please leave an open issue with some details about
+why the feedstock was archived (e.g. it has been deprecated by a new feedstock),
+and link it in your PR description.
+
+Conversely, if you want to request a feedstock to be unarchived, follow the same steps but for the `unarchive/` directory.
```
Comment (beckermr) [inline README.md:53]: You can't leave new issues on archived feedstocks. This comments seems to suggest the user should?

Comment (jaimergp) [reply README.md:53]: I meant to say the user should open one _before_ the PR for archival is merged (which should still be possible, right?)

Comment (jaimergp) [reply README.md:53]: e.g. https://github.com/conda-forge/retworkx-feedstock/issues/8

Comment (beckermr) [reply README.md:53]: Yes. I was just referring to unarchiving.

Comment (jaimergp) [reply README.md:53]: Ah, I see, I'll rephrase for clarity.

Comment (jaimergp) [reply README.md]: Done.

**Review by beckermr — APPROVED** [review]

**Comment by beckermr** [comment]
thank you @jaimergp!

**Comment by jakirkham** [comment]
Thanks Jaime and Matt! 🙏

**Comment by hmaarrfk** [comment]
Superb!

**Comment by jakirkham** [comment]
Updating our docs to reference this ( https://github.com/conda-forge/conda-forge.github.io/pull/1870 ), please take a look 🙂

**Comment by beckermr** [comment]
thank you!

**Comment by jakirkham** [comment]
Testing this out in PR ( https://github.com/conda-forge/admin-requests/pull/536 )

**Pull request merged**